packages/test/test-version-utils/src/describeCompat.ts

```diff
@@ -204,7 +209,7 @@ export interface ITestObjectProviderOptions {
  */
 export type DescribeCompatSuite = (
```

Comment:
Can you update an existing end-to-end test (or add a new one) that uses this new pattern. It would be helpful to see it in action and review it from that perspective as well.